### ISSUE TITLE
Use avatarify with Fujifilm X Webcam

### ISSUE THREAD

**Issue by julesfromspace** [alievk/avatarify#231]
I am using a Fujifilm camera as webcam with Fujifilms X webcam tool which enables to stream the video signal coming from a Fujifilm camera to e.g. Zoom, vMix. 

Unfortunately I don't get a video signal in the cam window of avatarify. Does this have to do with the fact the Fujifilm camera is not a regular webcam? Is there a way around this? Already tried by chaning the set CAMID number in the settings_windows file ...

**Comment by alievk** [comment]
But there is no `CAMID` in `settings_windows` file..

**Comment by julesfromspace** [comment]
True, but I followed your video tutorial and there you set a CAMID. The camera was not detected automatically that's why I tried the CAMIDs ...
